sdks/python/apache_beam/typehints/typehints.py

```diff
@@ -1186,20 +1186,18 @@ def __getitem__(self, type_params):
 
 def normalize(x, none_as_type=False):
   # None is inconsistantly used for Any, unknown, or NoneType.
+
+  # Avoid circular imports
+  from apache_beam.typehints import native_type_compatibility
+
+  if sys.version_info >= (3, 9) and isinstance(type, types.GenericAlias):
+    x = native_type_compatibility.convert_builtin_to_typing(x)
```

Comment:
Based on the pre-commit failures it seems like this conversion isn't happening

Reply:
`isinstance(x, types.GenericAlias)` instead of `isinstance(type, types.GenericAlias)`